README.md

```diff
@@ -100,31 +94,51 @@ should work on LX-branded zones as well.
     deployment (see
     [the CNS operator guide](https://github.com/joyent/triton-cns/blob/master/docs/operator-guide.md)).
     We'll assume for the sake of example here that the CNS suffix for the
-    DC is `us-west-1.triton.zone`.
+    DC is
+    ```
+    us-west-1.triton.zone
+    ```
  2. Find the CNS-generated name for your container. One way to do this is
-    to look at the output of `triton inst get <instance>` for the `dns_names`
-    array. As an example, let's consider
-    `blog.svc.3c330096-89e6-11e7-9f13-23d71a63353e.us-west-1.triton.zone`.
+    to look for the `dns_names` array in the output of
+    ```shell
+    triton inst get <instance>
+    ```
+    As an example, let's consider
+    ```
+    blog.svc.3c330096-89e6-11e7-9f13-23d71a63353e.us-west-1.triton.zone
+    ```
  3. Set up your desired DNS name as a CNAME to this CNS-generated name. If you
     are hosting the root of your domain, it's also fine to just set up a
     regular A record instead, as long as you also deploy a TXT record
-    containing the full UUID of the container. We'll use `blog.example.com`
+    containing the full UUID of the container. We'll use
+    ```
+    blog.example.com
+    ```
     and CNAME it to
-    `blog.svc.3c330096-89e6-11e7-9f13-23d71a63353e.us-west-1.triton.zone`.
+    ```
+    blog.svc.3c330096-89e6-11e7-9f13-23d71a63353e.us-west-1.triton.zone
+    ```
  4. Set up `_acme-challenge.<domain>` as a CNAME to
     `_acme-challenge.<cnsdomain>`. We'll set up
-    `_acme-challenge.blog.example.com` as a CNAME to
-    `_acme-challenge.blog.svc.3c330096-89e6-11e7-9f13-23d71a63353e.us-west-1.triton.zone`.
+    ```
+    _acme-challenge.blog.example.com
+    ```
+    as a CNAME to
+    ```
+    _acme-challenge.blog.svc.3c330096-89e6-11e7-9f13-23d71a63353e.us-west-1.triton.zone
+    ```
  5. Inside the container, download and extract the `dehydrated.tar.gz` file
     from the [latest GitHub release](https://github.com/joyent/triton-dehydrated/releases/)
     into a directory.
  6. Create a new file `domains.txt` in the directory containing just one line
-    with the full domain name you want on the certificate (e.g.
-    `blog.example.com`).
- 7. Register with the **Let's Encrypt** server by running
-    `./dehydrated --register --accept-terms`
- 8. Get the first certificate by running
-    `./dehydrated -c`
+    with the full domain name you want on the certificate, e.g.
+    ```
+    blog.example.com
+    ```
+ 7. Get the first certificate by running
+    ```shell
+    ./dehydrated -c --accept-terms
+    ```
 
 Now you will find your certificate files in `./certs/blog.example.com/`. You
 should configure your webserver to get the private key and certificate file
```

Comment:
The changes to the readme seem to be cosmetic, and also not formatted to render properly. I'm inclined to recommend reverting the entire file.